addOns/automation/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 - Warning if unexpected element under `activeScan` job (e.g. misplaced `rules`).
 - Help details warning against specifying default ports (80/443) (Issue 7649).
+- Method to get running plans - can be used by scripts to interact with the plans.
```

Comment:
This needs to be moved up to the Unreleased version.

Reply:
Doh!